virttest/utils_test/libvirt.py

```diff
@@ -3577,3 +3578,26 @@ def get_disk_alias(vm, source_file=None):
                 logging.info("Ignore error of source attr getting for file: %s" % e)
                 pass
     return None
+
+
+def check_cmd_expected(check_cmd, matched, is_expected):
+    """
+    check the cmd result that process run
+
+    Params check_cmd: the command need to run
+    Params matched: need to match or not match result
+    Params is_expected:
+    """
```

Comment:
Format need update and "is_expected" need to explain

Reply:
And better give the type for it, boolean or str

Reply:
add type to commond